src/test/resources/bundles/userCardExamples2/template/fr/question.handlebars

```diff
@@ -68,6 +70,13 @@
     }
     loadEntitiesList();
 
+    function loadEntityUsedForResponse() {
+        let responseEntityDiv = document.getElementById("response-entity-div");
+        let responseEntity = templateGateway.getEntityUsedForUserResponse();
+        let userAllowedToRespond = templateGateway.isUserAllowedToRespond();
+        if (userAllowedToRespond && responseEntity) responseEntityDiv.innerHTML = "Entité utilisée par l'utilisateur pour répondre : " + responseEntity;
```

Comment:
Show the name of the entity as for entites allowed to respond (use templateGateway.getEntityName)

Reply:
Done